packages/wonder-blocks-modal/components/one-column-modal.js

```diff
@@ -89,10 +93,12 @@ class ContentWrapper extends React.Component<Props> {
     }
 }
 
+const WrappedContentWrapper = MediaLayoutWrapper(ContentWrapper);
```

Comment:
Having an explicit `MediaConsumer` component that uses function-as-child would mean that we wouldn't have to wrap everything with `MediaLayoutWrapper`.  Things can get a little dicey when you have to use multiple wrappers on the same component too since sometimes they have to be applied in a specific order.  I'll tackle this in the provider/consumer PR that I've proposed.

Reply:
Also, in talking with other devs, people in general preferred the FaC pattern as opposed to the HoC pattern.